uber/templates/preregistration/preregbase.html

```diff
@@ -64,12 +66,12 @@
             row: '#badge-types',
             selector: '.badge-type-selector',
             options: [{
```

Comment:
Hmm, I do like that idea for the confirm page, but this might create some confusing situations for the main prereg form.  Suppose we get the following situation:
- Someone starts filling out the prereg form; they leave the default "Attendee" badge type selected
- They click the "Sign me up" box for volunteering and submit the form
- Because they didn't fill out all of the required fields, they're taken back to the form, but now the badge type option says "Volunteer"

The same thing can happen if someone clicks "Edit" before paying.

I do like the idea of showing more specific info about their registration on other places though, so maybe we can change this to say like
``{% if c.PAGE_PATH == '/preregistration/form` %}Attending{% else %}{{ attendee.ribbon_and_or_badge }}{% endif %}``

How does that sound?